clients/githubrepo/client.go

```diff
@@ -55,13 +62,25 @@ func (client *Client) GetFileContent(filename string) ([]byte, error) {
 	return client.tarball.getFileContent(filename)
 }
 
+func (client *Client) ListMergedPRs() []clients.PullRequest {
```

Comment:
shall we have these function return `error` as well? Same question for `GetDefaultBranch `

Reply:
Sure, done.